qiskit/backends/ibmq/ibmqprovider.py

```diff
@@ -191,29 +187,33 @@ def list_accounts(self):
 
         return information
 
-    def load_accounts(self):
-        """Load all accounts found in the system.
+    def load_accounts(self, **kwargs):
+        """Load IBMQ accounts found in the system, subject to optional filtering.
 
-        Automatically load all the accounts found in the system. This method
+        Automatically load the accounts found in the system. This method
         looks for credentials in the following locations, in order, and
-        returning as soon as credentials are found::
+        returns as soon as credentials are found:
 
         1. in the `Qconfig.py` file in the current working directory.
         2. in the environment variables.
         3. in the `qiskitrc` configuration file
 
         Raises:
-            IBMQAccountError: if there already loaded accounts in the session, or
-                no credentials could be found.
+            IBMQAccountError: if attempting to load previously loaded accounts,
+                    or if no credentials can be found.
         """
-        if self._accounts:
```

Comment:
I suggest we give a warning here.  Just so the user knows that the call did nothing due to the accounts list being populated.

Reply:
that was the old behavior. here, the account list gets appended with whatever account that matches the filter (even if you had loaded stuff previously).